src/EnergyPlus/SingleDuct.cc

```diff
@@ -2747,7 +2747,7 @@ namespace SingleDuct {
 		UpdateSys( SysNum );
 
 		// At the current air mass flow rate, calculate heating coil load
-		QActualHeating = QToHeatSetPt - MassFlow * CpAirZn * ( SysInlet( SysNum ).AirTemp - ZoneTemp ); // reheat needed
+		QActualHeating = QToHeatSetPt - MassFlow * ( CpAirSysIn * SysInlet( SysNum ).AirTemp - CpAirZn * ZoneTemp ); // reheat needed
```

Comment:
This is very different from how coil models calculate Q. This is Mdot * ( delta CpT) whereas the coil models are Mdot * Cpin * dT. So why the difference? No reason to hold this up, just thinking globally.